debian/control

```diff
@@ -19,7 +19,7 @@ Description: An open source re-implementation of RollerCoaster Tycoon 2.
  The gameplay revolves around building and maintaining an amusement park
  containing attractions, shops and facilities. The player must try to make a
  profit and maintain a good park reputation whilst keeping the guests happy.
- 
+ .
```

Comment:
Well is this 
![image](https://github.com/OpenRCT2/OpenRCT2/assets/550290/fb6fb9fa-ff3b-4901-9afe-96131a4b417c)
 all over again?

Reply:
Luckily the Debian package description is not connecting to sandboxing working or not 😄

Reply:
So we're told ;)